src/main/java/org/springframework/hateoas/Link.java

```diff
@@ -263,7 +448,34 @@ public static Link valueOf(String element) {
 				throw new IllegalArgumentException("Link does not provide a rel attribute!");
 			}
 
-			return new Link(matcher.group(1), attributes.get("rel"));
+			Set<String> unrecognizedHeaders = unrecognizedHeaders(attributes);

```

Comment:
Following Postel's Law I think we should not throw an exception here but just ignore attributes that we don't understand.